lametro/templates/partials/person_bio_form.html

```diff
@@ -0,0 +1,16 @@
+{% load extras %}
+{% load lametro_extras %}
+
+<form role="form" method="POST" enctype="multipart/form-data">
+    {% csrf_token %}
+
+    {% if form2.biography.errors %}
+        <br>
+        <p style="color: #eb6864;">{{form2.biography.errors.as_text}}</p>
+    {% endif %}
+    {{form2.bio_form}}
+    <label for="{{form2.biography.id_for_label}}">Update {{form2.biography.label}}:</label>
+    {{form2.biography}}
+    <input type="submit" value="Submit">
```

Comment:
Can you use `button` elements for submit?

Reply:
Done! Is using a button for submissions more of a better practice?